google-cloud-firestore/Rakefile

```diff
@@ -0,0 +1,230 @@
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
GAX boilerplate now reads: "Google LLC All rights reserved."

Reply:
I opened GoogleCloudPlatform/google-cloud-ruby#1882 for this.